packages/dd-trace/src/util.js

```diff
@@ -10,6 +10,12 @@ function isFalse (str) {
   return str === 'false' || str === '0'
 }
 
+function isTrueOrFalse (str) {
```

Comment:
Why is this function needed?

Reply:
Because the config for AppSec is not just a boolean now, it can be in three different states;
- true
- false
- {not set}

It is that way because we only want to enable remote config when it is not set (the local config takes precedence).
Maybe you have a better idea to implement this three state check ?

Reply:
Do you mean there are 3 final states? What are they? Can remote config be partially enabled?

Reply:
This is what I'm trying to implement:
![image](https://user-images.githubusercontent.com/70517128/199511989-2ef74d9f-044b-4c19-9810-f24ca600e675.png)
As you can see the local conf (`DD_APPSEC_ENABLED`) can be in three different states: `true`, `false`, and "not set".
We only activate remote appsec activation if `DD_APPSEC_ENABLED` is "not set".
This is why I need `isTrueOrFalse()`, I need to differentiate between truthy, falsy, and undefined-ish.

Reply:
Look at this line: https://github.com/DataDog/dd-trace-js/pull/2401/files#diff-dae9ab64bee2d029cc08e9fdb70b3cd9d2f897e9b451d325e51033d457bb2994R54
This undefined value is sourced by "isTrueOrFalse" when the config is neither explicitely true or false, undefined means "not locally set". We can zoom if you still think we can do this otherwise.

Reply:
Alright I've changed it to `appsec_enabled && isTrue(appsec_enabled)` and it looks like it's working. I thought I previously tried it but I guess I was wrong